ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml

```diff
@@ -6,11 +6,24 @@ $defs:
   adapter_config:
     type: dict
     keys:
+      endpoint_ports:
```

Comment:
endpoint ports is already part of the connected endpoints schema, which inherits from this one. We only have it there, because we _don't_ want it under `network_ports` which _also_ inherits from this one.
Same for `descriptions`, I don't think it fits well with the `network_ports` use case of batch configuration of larger ranges, so I suggest that you move the `descriptions` key to `defs_connected_endpoints.schema.yml` and remove the `endpoint_ports` from here. You may want to update the description of `endpoint_ports` in that file instead.